docs/_static/clp-versions.json

```diff
@@ -1,4 +1,8 @@
 [
+  {
+    "version": "0.1.1",
+    "url": "https://docs.yscope.com/clp/v0.1.1/"
```

Comment:
I can't open the link but I assume once the change is in, the link will be accessible?

Reply:
After the change is merged and we publish the latest version, yep.